app/assets/stylesheets/assignments.scss

```diff
@@ -106,13 +106,13 @@ input[type='number'] {
 }
 
 .block {
-  @include border-radius(5px);
+  border-radius: $radius;
   border: 1px solid $blue;
   margin: 0 auto 1.5em;
   padding: 0;
 
   h2 {
-    @include border-radius-top(5px);
+    border-radius: $radius $radius 0 0;
```

Comment:
Properties should be ordered background, border-bottom, border-radius, color, font-size, font-weight, margin, padding